doc/content/getting-started/api/_index.md

```diff
@@ -17,25 +17,32 @@ A complete list of API endpoints is available in the [API Reference]({{< ref "re
 Additional fields may be specified in HTTP requests by appending them as query string parameters. For example, to request the `name`, `description`, and `locations` of devices in an `EndDeviceRegistry.Get` request, add these fields to the `field_mask` field. To get this data for device `dev1` in application `app1`:
 
 ```bash
-$ curl --location --header "Authorization: Bearer NNSXS.XXXXXXXXX" --header 'Accept: application/json' \
-https://thethings.example.com/api/v3/applications/app1/devices/dev1?field_mask=name,description,locations
+$ curl --location \
+  --header "Authorization: Bearer NNSXS.XXXXXXXXX" \
+  --header 'Accept: application/json' \
+  'https://thethings.example.com/api/v3/applications/app1/devices/dev1?field_mask=name,description,locations'
 ```
 
 The same request in tenant `tenant1` on a multi-tenant deployment:
 
 ```bash
-$ curl --location --header "Authorization: Bearer NNSXS.XXXXXXXXX" --header 'Accept: application/json' \
-https://tenant1.thethings.example.com/api/v3/applications/app1/devices/dev1?field_mask=name,description,locations
+$ curl --location \
+  --header "Authorization: Bearer NNSXS.XXXXXXXXX" \
+  --header 'Accept: application/json' \
+  'https://tenant1.thethings.example.com/api/v3/applications/app1/devices/dev1?field_mask=name,description,locations'
 ```
 
 Fields may also be specified as a JSON object in a POST request.
 
 To get a stream of events for device `dev1` in application `app1` :
 
 ```bash
-$ curl --location --header 'Authorization: Bearer NNSXS.XXXXXXXXX' --header 'Accept: application/json' \
-     --header 'Content-Type: application/json' --request POST 'https://thethings.example.com/api/v3/events' \
-     --data-raw '{
+$ curl --location \
+  --header 'Authorization: Bearer NNSXS.XXXXXXXXX' \
+  --header 'Accept: application/json' \
+  --header 'Content-Type: application/json' \
+  --request POST 'https://thethings.example.com/api/v3/events' \
+  --data-raw '{
```

Comment:
Does this work; order of flags, url, more flags?

Reply:
My help says;

```
Usage: curl [options...] <url>
```

Reply:
it does work but it's semantically wrong, i'll rewrite